src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/MakeAmendmentOfSpecificMethodologyAuthorizationHandlerTests.cs

```diff
@@ -29,21 +31,25 @@ public class MakeAmendmentOfSpecificMethodologyAuthorizationHandlerTests
 
         private static readonly Methodology PublicMethodology = new Methodology
         {
+            Id = Guid.NewGuid(),
             Status = Approved,
-            PublishingStrategy = MethodologyPublishingStrategy.Immediately,
+            PublishingStrategy = Immediately,
             MethodologyParent = new MethodologyParent()
         };
 
         private static readonly Methodology PrivateMethodology = new Methodology
         {
+            Id = Guid.NewGuid(),
```

Comment:
I thought it might be a bit confusing declaring them without a status value, or leaving a status value which didn't agree with the value being returned by the `IsPubliclyAccessible` mocked method in the various tests.

I've changed it to what you said now, and gone for no status field.

Reply:
Ahh gotcha.  Yeah happy either way then, but happy if you're happy to keep the changed version